### PR TITLE
[OTA-3093] Messages, Tooltip and Wording

### PR DESCRIPTION
**Pull request by piotrcichonmee**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/style/ota-default/_campaign-wizard.scss [hunk 1]
```diff
@@ -223,6 +223,17 @@
         .distribution-settings {
           padding: 15px 0;
 
+          .consent {
+            color: #000000;
+            font-weight: 400;
+            line-height: 14px;
+          }
+          .warning {
+            color: #6F737A;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/style/ota-default/_campaign-wizard.scss]: `$campaign-cancelled-color`

File: ota-plus-web/app/reactapp/style/ota-default/_campaign-wizard.scss [hunk 2]
```diff
@@ -510,14 +521,18 @@
...
-          margin-top: 20px;
           padding: 0 20px;
 
           .title {
             font-weight: bold;
             font-size: 18px;
+            margin-top: 20px;
+          }
+          .warning {
+            color: #6F737A;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/style/ota-default/_campaign-wizard.scss]: `$campaign-cancelled-color`

File: ota-plus-web/app/reactapp/style/ota-default/common.scss [hunk 3]
```diff
@@ -375,6 +375,30 @@ strong {
...
+.anim-info-container {
+  width: 768px;
+  margin-left: -24px;
+  &.member-adding {
+    width: auto;
+    margin: -10px 0 10px 0;
+  }
+  .anim-info {
+    height: 40px;
+    background-color: #48DAD0;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/style/ota-default/common.scss]: `$color-mint`

File: ota-plus-web/app/reactapp/style/ota-default/common.scss [hunk 4]
```diff
@@ -375,6 +375,30 @@ strong {
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    &.member-adding {
+      width: auto;
+    }
+    .text {
+      color: #FFFFFF;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/style/ota-default/common.scss]: `$color-white`

File: ota-plus-web/app/reactapp/style/ota-default/common.scss [hunk 5]
```diff
@@ -413,6 +437,14 @@ strong {
     padding: 5px 15px;
     border: 1px solid;
   }
+  &.file-input {
+    border: 0px;
+    color: #00afaa;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/style/ota-default/common.scss]: `$switch-blue-color`

File: ota-plus-web/app/reactapp/src/partials/OperationCompletedInfo.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/partials/OperationCompletedInfo.js]: Please keep up file naming convention throughout the app (`.jsx` for components)

File: ota-plus-web/app/reactapp/style/ota-default/modals.scss [hunk 7]
```diff
@@ -737,10 +745,24 @@
   .ecu-types-select {
     margin-top: 10px;
   }
-
   .upload-wrapper {
-    margin: 20px 0 0 0;
+    display: inline-block;
+    margin: 10px 0 0 0;
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/style/ota-default/modals.scss]: `margin: 10px 0 0;`

File: ota-plus-web/app/reactapp/style/ota-default/pages/_page-profile.scss [hunk 8]
```diff
@@ -92,12 +92,14 @@
...
+      margin-right: 20px;
     }
 
     .description {
       color: $dashboard-header-text;
-      font-size: 14px;
-      margin: 0 0 10px 36px;
+      font-size: 11px;
+      margin: 0 20px 10px 36px;
+      white-space: pre-line
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/style/ota-default/pages/_page-profile.scss]: ;

File: ota-plus-web/app/reactapp/src/partials/TimePicker.jsx [hunk 9]
```diff
@@ -61,15 +61,23 @@ class TimePicker extends Component {
         ))}
       </select>
     ) : null);
-
+    console.log('TimePicker - render - this.selectedType: ', this.selectedType);
+    console.log('TimePicker - render - times[0]: ', times[0]);
+    console.log('TimePicker - render - times[1]: ', times[1]);
```
[the remark this answers is not in the document]
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/partials/TimePicker.jsx]: Done

File: ota-plus-web/app/reactapp/style/ota-default/_campaign-wizard.scss (hunk 1 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/style/ota-default/_campaign-wizard.scss]: Done

File: ota-plus-web/app/reactapp/style/ota-default/_campaign-wizard.scss (hunk 2 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/style/ota-default/_campaign-wizard.scss]: Done

File: ota-plus-web/app/reactapp/style/ota-default/common.scss (hunk 3 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/style/ota-default/common.scss]: Done

File: ota-plus-web/app/reactapp/style/ota-default/common.scss (hunk 4 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/style/ota-default/common.scss]: Done

File: ota-plus-web/app/reactapp/style/ota-default/common.scss (hunk 5 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/style/ota-default/common.scss]: Done

File: ota-plus-web/app/reactapp/style/ota-default/modals.scss (hunk 7 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/style/ota-default/modals.scss]: Done

File: ota-plus-web/app/reactapp/style/ota-default/pages/_page-profile.scss (hunk 8 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/style/ota-default/pages/_page-profile.scss]: Done

**Pull request merged**